pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -274,8 +274,14 @@ type DiscordConfig struct {
 	// The secret's key that contains the Discord webhook URL.
 	// The secret needs to be in the same namespace as the AlertmanagerConfig
 	// object and accessible by the Prometheus Operator.
-	// +required
-	APIURL v1.SecretKeySelector `json:"apiURL"`
+	// Either `apiURL` or `webhookURLFile` is required.
+	// +optional
+	WebhookURL *v1.SecretKeySelector `json:"apiURL,omitempty"`
```

Comment:
should it be webhookurl?

Reply:
Thanks for your review.
ref: 
https://github.com/prometheus-operator/prometheus-operator/blob/8df0ff1baf8e003f18ca224e58306a1c28ca5d16/example/prometheus-operator-crd-full/monitoring.coreos.com_alertmanagerconfigs.yaml#L219-L242

still `apiURL` in CRD file.